client/src/components/CompaniesTable.tsx

```diff
@@ -2,15 +2,16 @@ import * as React from "react";
 import styled from "styled-components";
 import MaterialTable, { Column } from "material-table";
 import { ReactComponent as ImportIcon } from "./resources/file-upload.svg";
+import ATable, { TableState, TableProps } from "./ATable";
 
 export interface CompaniesTableProps {
   uploadHandler: Function;
 }
 
-export interface CompaniesTableState {
-  companies: DisplayCompany[];
-  columns: Array<Column<DisplayCompany>>;
-}
+// export interface CompaniesTableState {
```

Comment:
Please don't leave commented out code, if you need access to it you can use the git history